legal-api/src/legal_api/resources/v2/request_tracker.py

```diff
@@ -30,11 +31,13 @@
 @jwt.has_one_of_roles([UserRoles.admin_edit, UserRoles.bn_edit])
 def get_bn_request_trackers(identifier: str):
     """Return a list of request trackers."""
-    legal_entity = LegalEntity.find_by_identifier(identifier)
-    if legal_entity is None:
+    business = business_service.fetch_business(identifier)
+    if business is None:
         return ({"message": "A valid business is required."}, HTTPStatus.BAD_REQUEST)
 
-    request_trackers = RequestTracker.find_by(legal_entity.id, RequestTracker.ServiceName.BN_HUB)
+    request_trackers = RequestTracker.find_by(
+        business.id, RequestTracker.ServiceName.BN_HUB, None, None, None, business
```

Comment:
remove `business.id` param after removing this param from `RequestTracker.find_by`

Reply:
fixed